.mailmap

```diff
@@ -251,6 +251,8 @@ Alex Lubbock <code@alexlubbock.com>
 Alex Malins <github@alexmalins.com>
 Alex Meiburg <timeroot.alex@gmail.com>
 AlexCQY <alex_chua@u.nus.edu>
+Alexander Behrens <Alex031544@users.noreply.github.com>
+Alexander Behrens <alex.git@gmx.net> Alex031544 <alex.git@gmx.net>
```

Comment:
This should not be two separate entries:
https://github.com/sympy/sympy/blob/4165a5016bd8708ee83aea11a22e6217fb38800b/.mailmap#L139-L145

Reply:
Thank you. Fixed.